lnrpc/wtclientrpc/wtclient.proto

```diff
@@ -0,0 +1,137 @@
+syntax = "proto3";
+
+package wtclientrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/wtclientrpc";
+
+message AddTowerRequest {
+    // The identifying public key of the watchtower to add.
+    bytes pub_key = 1 [json_name = "pub_key"];
+
+    // A network address the watchtower is reachable over.
+    string address = 2 [json_name = "address"];
+}
+
+message AddTowerResponse {
+}
+
+message RemoveTowerRequest {
+    // The identifying public key of the watchtower to remove.
+    bytes pub_key = 1 [json_name = "pub_key"];
+
+    /*
+    If set, then the record for this address will be removed, indicating that is
+    is stale. Otherwise, the watchtower will no longer be used for future
+    session negotiations and backups.
+    */
+    string address = 2 [json_name = "address"];
+}
+
+message RemoveTowerResponse {
+}
+
+message GetTowerInfoRequest {
+    // The identifying public key of the watchtower to retrieve information for.
+    bytes pub_key = 1 [json_name = "pub_key"];
+}
+
+message TowerSession {
+    /*
+    The total number of successful backups that have been made to the
+    watchtower.
+    */
+    uint32 num_backups = 1 [json_name = "num_backups"];
+
+    /*
+    The total number of backups that are currently pending to be acknowledged by
+    the watchtower.
+    */
+    uint32 num_pending_backups = 2 [json_name = "num_pending_backups"];
+
+    // The total number of backups allowed by the watchtower.
+    uint32 num_max_backups = 3 [json_name = "num_max_backups"];
```

Comment:
perhaps just `max_backups`?

Reply:
Seems better to leave as is to maintain consistency with the other fields?

Reply:
the other two are dynamic, while this is static. was thinking just calling it `max_updates` would differentiate, but not a blocker

Reply:
Done.